tests/test_switch_server.c

```diff
@@ -1032,7 +1032,7 @@ testclient_run (gpointer data)
   client->mainloop = g_main_loop_new (NULL, TRUE);
 
   connect_ok =
-      gst_switch_client_connect (GST_SWITCH_CLIENT (client), CLIENT_ROLE_UI, "tcp:host=127.0.0.1,port=5000");
+      gst_switch_client_connect (GST_SWITCH_CLIENT (client), "tcp:host=127.0.0.1,port=5000");
```

Comment:
I think it would be good if clients still advertised their "type" or "role"? This would be useful for logging?